x-pack/legacy/plugins/integrations_manager/server/index.ts

```diff
@@ -3,8 +3,12 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import { Plugin, PluginInitializerContext } from './plugin';
+import { Plugin, IntegrationsManagerPluginInitializerContext } from './plugin';
 
-export function plugin(initializerContext: PluginInitializerContext) {
+// Kibana NP needs config to be exported from here, see https://github.com/elastic/kibana/pull/45299/files#r323254805
+import { config } from './config';
+export { config };
```

Comment:
```suggestion
export { config } from './config';
```

Reply:
If we're not using `config` in the file we can use `export { config } from './config';`